### DISCUSSION TITLE
Copilot.vim not working in YAML files by default

### DISCUSSION THREAD

**Discussion by dharmab** [community/community#108329] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot.vim disables suggestions in YAML files by default, while Visual Studio Code enables it by default. This was [astonishing behavior.](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)

Workaround: Add this line to `.vimrc`:

```vimscript
let g:copilot_filetypes = {'yaml': v:true}
```

Expected behavior: The default filetypes are the same for all editors.

**Comment by tpope** [comment, marked as answer]
This is now fixed, thanks. The reason it was off by default is that VS Code _used to_ have it off by default too.
